processor/configuration/src/test/java/com/redhat/service/smartevents/processor/actions/slack/SlackActionValidatorTest.java

```diff
@@ -20,6 +20,19 @@ class SlackActionValidatorTest {
     @Inject
     SlackActionValidator validator;
 
+    @Test
+    void isInvalidWithNoParameters() {
+        Map<String, String> params = new HashMap<>();
```

Comment:
FYI @kostola In similar cases, we can use just `Collections.emptyMap();`

https://stackoverflow.com/questions/14846920/collections-emptymap-vs-new-hashmap